src/styles/_achievementcontrol.scss

```diff
@@ -0,0 +1,384 @@
+.AchievementControl {
+  display: flex;
+  flex-direction: row;
+  flex: 1 1 100%;
+  background-size: cover;
+  background-image: linear-gradient(
+      to right,
+      rgba(0, 0, 0, 0.3),
+      rgba(0, 0, 0, 0.6),
+      rgba(0, 0, 0, 0.3)
+    ),
+    url('https://source-academy-assets.s3-ap-southeast-1.amazonaws.com/locations/galaxy/normal%40x2.png');
```

Comment:
These assets should be committed into public/assets, or the base URL should be an SCSS variable derived from an environment variable (and all references to the assets S3 bucket incl. those in the game should be based off this env. variable)

In other words don't hardcode the S3 bucket URL

Reply:
Done https://github.com/source-academy/cadet-frontend/commit/74e2652eb16a58e559e5c1b42e976b0a04626d93